README.md

```diff
@@ -1 +1,5 @@
-# Review-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Nice work in this part, just try to set more information on the Readme file about the project and the technologies used on it, remember that this information will be useful for all those developers who want to use your reference project, you can use [the microverse template](https://github.com/microverseinc/readme-template),